sql/parser/builtins.go

```diff
@@ -382,8 +382,8 @@ var builtins = map[string][]builtin{
 			types:      typeList{timestampType},
 			returnType: DummyInterval,
 			impure:     true,
```

Comment:
So purity is defined within the scope of a statement? I'd like to update the comment.